deploy/docker/README.md

```diff
@@ -149,7 +173,7 @@ Assume you are using redis auth plugin, for example:
 
 docker run -d --name emqx -p 18083:18083 -p 1883:1883 -p 4369:4369 \
     -e EMQX_LISTENER__TCP__EXTERNAL=1883 \
-    -e EMQX_LOADED_PLUGINS="emqx_auth_redis,emqx_recon,emqx_retainer,emqx_management,emqx_dashboard" \
+    -e EMQX_LOADED_PLUGINS="emqx_auth_redis" \
```

Comment:
just a q: why delete ?

Reply:
> just a q: why delete ?

Previously, when users used `EMQX_LOADED_PLUGINS` to load plug-ins, they must manually specify `emqx_recon, emqx_retainer, emqx_management, emqx_dashboard`, otherwise these plug-ins cannot start normally and will cause emqx to fail to start. 

Now regardless of whether the user specifies these plug-ins, these plug-ins will work normally start up